package.json

```
@@ -27,10 +27,10 @@
   },
   "homepage": "https://github.com/dashevo/DashJS#readme",
   "dependencies": {
-    "@dashevo/dapi-client": "~0.8.0-dev.14",
-    "@dashevo/dashcore-lib": "~0.18.0",
-    "@dashevo/dpp": "0.10.0-dev.14",
-    "@dashevo/wallet-lib": "5.0.3",
+    "@dashevo/dapi-client": "~0.9.0-dev.6",
+    "@dashevo/dashcore-lib": "^0.18.0",
+    "@dashevo/dpp": "^0.11.0-dev.4",
```

@Alex-Werner [0.11.0](https://github.com/dashevo/js-dpp/tree/v0.11.0) was released. Can/should we use that instead of the dev branch?
```suggestion
    "@dashevo/dpp": "^0.11.0",
```